app/Rules/SettingCaptchaQuestions.php

```diff
@@ -0,0 +1,60 @@
+<?php
```

Comment:
Should we also have a test case for this Rule. I think it'd be pretty straight forward to set up and will help future generations in understanding what rules a set of questions is expected to adhere to.

Reply:
I was torn on whether or not the test coverage for the controller is sufficient or if it makes sense to also add a test for the rule. I could definitely be convinced but as I was writing tests for the rule it felt mostly like I was copy/pasting the controller tests so I've omitted it for now